app/pandas.ipynb

```diff
@@ -391,9 +514,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# use ** operator here"
-   ],
+   "source": "exponentiation = s1 ** s2",
```

Comment:
The exponentiation operation is defined but not executed. You should print the result to verify the operation.